topi/python/topi/cuda/conv2d_int8.py

```diff
@@ -250,18 +241,11 @@ def schedule_conv2d_NCHWc_int8(cfg, s, output):
 
     # cooperative fetching
     for load in [AA, WW]:
-        if load == AA:
-            n, f, y, x, c = s[load].op.axis
-            if pad_data == packed_data and stride_h == 1 and stride_w == 1:
-                s[load].vectorize(c)
-                fused = s[load].fuse(n, f, y, x)
-            else:
-                c, _ = s[load].split(c, factor=4)
-                fused = s[load].fuse(n, f, y, x, c)
-        else:
-            n, f, y, x, oc_chunk, c = s[load].op.axis
-            fused = s[load].fuse(n, f, y, x, oc_chunk)
-            s[load].vectorize(c)
+        c = s[load].op.axis[-1]
+        c_outer, c = s[load].split(c, factor=4)
+        s[load].vectorize(c)
+        fused = s[load].op.axis[:-1] + [c_outer]
```

Comment:
s[load].op.axis[:-1] also doesn't give me correct output

Reply:
@Laurawly Could you elaborate? The idea is to fuse all outer axis, except the inner c

Reply:
Yes, I tried on my local computer and it didn't work. Probably because I haven't rebated tvm to the latest version.